distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -553,13 +548,10 @@ private String createRegistryEntry(String id, String regId) throws Exception {
                 hasXPath("//TransactionResponse/TransactionSummary/totalDeleted",
                         CoreMatchers.is("0")));
 
-        XPath xPath = XPathFactory.newInstance()
-                .newXPath();
         String idPath = "//*[local-name()='identifier']/text()";
-        InputSource xml = new InputSource(IOUtils.toInputStream(response.getBody()
-                .asString(), StandardCharsets.UTF_8.name()));
-        String mcardId = xPath.compile(idPath)
-                .evaluate(xml);
+        String mcardId = XSearch.evaluate(idPath,
```

Comment:
In the interest of keeping things simple, I would prefer not to have a dependency on RESTAssured.